server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -242,6 +242,7 @@
     },
     "worker": {
         "alfredUrl": "http://alfred:3003",
+        "deltaStreamServerUrl": "http://localhost:3003",
```

Comment:
nit: remove `Server` in the `deltaStreamServerUrl` (-> `deltaStreamUrl`) name throughout this PR, because it is redundant. Also, it doesn't match the other naming conventions for Urls.